OKTA-736640 - Add new event type (#4921)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md

| **actor**                 |  The entity reporting the user risk change (can be a system principal, end user, or org administrator)                | Object        |         |
| type        | The type of actor object           | String     | User       |

## user risk detect
`user.risk.detect`

> **Description:** This event type indicates a user risk was detected. It can be used to monitor risk level detections for users. This event triggers when Okta detects that a user is associated with risk activity or context.

Reviewer: The descriptions of other event types seem to vary, but some use the present tense. This one should probably align with the style guide.

```suggestion
**Description:** Indicates a user risk is detected. Use it to monitor risk level detections for users. This event triggers when Okta detects that a user is associated with risk activity or context.
```

Author: Thanks, yes, this is a little clunky. Let me re-draft to match the existing event types.
